.github/workflows/rspec_tests.yml

```diff
@@ -14,6 +14,7 @@ jobs:
           - {os: ubuntu-18.04, ruby: 2.5}
           - {os: ubuntu-18.04, ruby: 2.6}
           - {os: ubuntu-18.04, ruby: 2.7}
+          - {os: ubuntu-18.04, ruby: 3.0}
```

Comment:
I assume we probably want to test this on Windows as well?

Reply:
okay never mind for now, looks like there are some issues with Ruby 3 + Windows that will take longer to sort out.